docs/tutorials/adding-a-module-to-your-runtime.md

```diff
@@ -448,20 +448,20 @@ Then inside the `testnet_genesis` function we need to add the contract configura
 fn testnet_genesis(initial_authorities: Vec<(AuraId, GrandpaId)>,
     root_key: AccountId,
     endowed_accounts: Vec<AccountId>,
-    enable_println: bool) -> GenesisConfig {
-    /*** Add this block ***/
+    _enable_println: bool) -> GenesisConfig {
```

Comment:
Why is this variable name preceded with an underscore?

I believe that's recommended for unused variables, but this one is used a few lines below.

Reply:
It is marked as unused by default. So I just took the definition that comes on the template. I can also point this out and make it part of the required changes.